core/resource/resource.go

```diff
@@ -243,7 +243,7 @@ func (r *Resource) MarkToCreate() error {
 
 func (r *Resource) MarkToUpdate() error {
 	switch r.status {
-	case StatusValidationSuccess, StatusUpdateFailure, StatusSuccess, StatusExistInStore:
+	case StatusValidationSuccess:
```

Comment:
Also for `to_update`, perhaps we can discuss about this as it's a bit different than the last discussion.

Reply:
Sure, let's have a discussion as the current implementation was wrong.